recipes/gridgen/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib"
+export CPPFLAGS="-I/$PREFIX/include $CPPFLAGS"
+
+pushd gridgen
+
+./configure --prefix=$PREFIX
+
+make
+make lib
+make shlib
```

Comment:
Alright, I trust you guys have spent enough time flailing around. :wink:
Just asking for my own edification. :smile: